CMakeLists.txt

```diff
@@ -53,6 +54,19 @@ else(MSVC)
   endif()
 endif(MSVC)
 
+if(USE_NNPACK)
+  find_package(NNPACK REQUIRED)
+  if(NNPACK_FOUND)
+    message(STATUS "Found NNPACK: ${NNPACK_INCLUDE_DIR}")
+    add_definitions(-DMXNET_USE_NNPACK=1)
+    include_directories(SYSTEM ${NNPACK_INCLUDE_DIR})
+    include_directories(SYSTEM ${NNPACK_INCLUDE_DIR}/../third-party/pthreadpool/include)
+    list(APPEND REQUIRED_LIBRARIES ${NNPACK_LIB})
+  else()
+    message(FATAL_ERROR "Can't find NNPACK. Please set NNPACK_FOUND, NNPACK_INCLUDE_DIR and NNPACK_LIB")
```

Comment:
Setting `NNPACK_ROOT` is optional I think, in case the `nnpack` library is only built in local workspace, and not yet installed in system directories. While `NNPACK_FOUND` is trying to report whether `nnpack` is found in system directories, like `/usr/local`.
Usually, after compiling `nnpack`, one should copy a list of files to `/usr/local` to make cmake script easily locate the header and library files. Here I list the required files:
- NNPACK/include/*
- NNPACK/lib/libnnpack.so or NNPACK/lib/libnnpack.a
- NNPACK/third-party/pthreadpool/include/pthreadpool.h